app/services/config-persistence/config.ts

```diff
@@ -61,16 +61,11 @@ export class ConfigPersistenceService extends PersistentStatefulService<IScenesC
     this.CLEAR_SCENES_COLLECTIONS();
     if (!fs.existsSync(this.configFileDirectory)) return;
 
-    const configsNames = fs.readdirSync(this.configFileDirectory).map(file => file.replace(/\.[^/.]+$/, ''));
-    const configsNamesWithoutBackup: string[] = [];
-
-    if (configsNames.length) {
-      configsNames.forEach(configName => {
-        if (configName.indexOf('_backup') === -1) configsNamesWithoutBackup.push(configName);
-      });
-      this.ADD_SCENES_COLLECTIONS(configsNamesWithoutBackup);
-    }
+    const configsNames = fs.readdirSync(this.configFileDirectory).filter(fileName => {
+      return fileName.indexOf('.bak') === -1;
```

Comment:
You should use a regex to make sure the `.bak` is at the end of the string:
```TypeScript
fileName.match(/\.bak$/)
```
This will return an array of match data if it matches, or `null` if it doesn't match.

Reply:
Done.